package.json

```diff
@@ -7,6 +7,13 @@
   "bin": {
     "pino": "./pretty.js"
   },
+  "files" : [
+    "pino.js", 
+    "browser.js",
+    "pretty.js",
+    "LICENEE",
```

Comment:
that's against the standard
besides - tests are 40K and that doubles the size of the package.
FYI.